mod_modular_vanilla/hooks/items/item.nut

```diff
@@ -126,5 +217,19 @@
 				delete this[p];
 			}
 		}
+
+		// MV: Part of MV_Variant framework for items
+		// We need a hookTree because various children of item.nut call their own
+		// updateVariant at the end of their own onDeserialize.
+		q.onDeserialize = @(__original) { function onDeserialize( _in )
+		{
+			__original(_in);
+			// We only apply our variant during deserialization if it is installed
+			// otherwise we let the original variant remain.
+			if (this.getFlags().has("MV_Variant") && this.__MV_isVariantInstalled())
```

Comment:
This first check should move into `__MV_isVariantInstalled` as it is a direct requirement for the following function `__MV_getFormattedVariantString` to not crash.

Also it makes a lot of sense, if `__MV_isVariantInstalled` returns `false` if the flag does not exist at all

Reply:
Eh.. your point makes sense. I think I put it here because doing this check inside `__MV_isVariantInstalled` will require me to put that check in all the child classes where I overwrite that function.